bot/seasons/easter/easter_riddle.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+import logging
+import random
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with open(Path('bot', 'resources', 'easter', 'easter_riddle.json'), 'r', encoding="utf8") as f:
+    RIDDLE_QUESTIONS = load(f)
+
+TIMELIMIT = 10
+
+
+class EasterRiddle(commands.Cog):
+    """This cog contains the command for the Easter quiz!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.quiz_messages = {}
+        self.winner = []
```

Comment:
Since this is a list of multiple winners I think it's more appropriate to name it `winners` (plural). Don't forget to update the docstring that mentions it on line 78.